CONTRIBUTING.md

```diff
@@ -113,6 +113,46 @@ the repository maintainers will integrate it, prepare changelogs, and
 ensure it gets released in one of upcoming Weekly releases.
 There is no additional action required from pull request authors at this point.
 
+### Pull requests management
+
+The project is using some labels to mark the status and the content of the pull requests.
+The complete list of labels can be found at https://github.com/jenkinsci/jenkins/labels.
+Out of those labels, here is a definition of labels the maintainers are using to manage the status of the pull requests:
+
+- `needs-docs` marks a pull requests as lacking documentation, either for developers (e.g., Javadoc) or users (e.g., changes the [Jenkins handbook](https://www.jenkins.io/doc/book/)).
```

Comment:
```suggestion
`needs-docs`:: Marks a pull requests as lacking documentation, either for developers (e.g., Javadoc) or users (e.g., changes to the [Jenkins handbook](https://www.jenkins.io/doc/book/)).
```

Reply:
this is a Markdown document, the `::` is not behaving as in a ascidoc document.